waku/v2/node/peer_manager/waku_peer_store.nim

```diff
@@ -143,3 +143,6 @@ proc selectPeer*(peerStore: PeerStore, proto: string): Option[RemotePeerInfo] =
 
 proc getPeersByDirection*(peerStore: PeerStore, direction: Direction): seq[StoredInfo] =
   return peerStore.peers().filterIt(it.direction == direction)
+
+proc getNotConnectedPeers*(peerStore: PeerStore): seq[StoredInfo] =
+  return peerStore.peers().filterIt(it.connectedness != Connected)
```

Comment:
Nit: You can remove the parenthesis from the `peers` property, as it is a noun.

```suggestion
  return peerStore.peers.filterIt(it.connectedness != Connected)
```

Reply:
sure fixed.